bert/src/tokenizer.rs

```diff
@@ -130,18 +135,12 @@ mod tests {
         let encoding = tokenizer
             .encode("These are normal, common EMBEDDINGS.")
             .unwrap();
-        assert_eq!(encoding.get_ids().len(), 258);
-        assert_eq!(
-            encoding.get_ids()[..15],
-            [0, 32255, 621, 3638, 4, 39210, 19515, 20090, 24057, 142_766, 5, 2, 1, 1, 1],
-        );
-        assert_eq!(
-            encoding.get_attention_mask()[..15],
-            [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 0, 0, 0],
-        );
+        assert_eq!(encoding.get_ids().len(), 12);
         assert_eq!(
-            encoding.get_type_ids()[..15],
-            [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+            encoding.get_ids(),
+            [101, 2122, 2024, 3671, 1010, 2691, 7861, 8270, 4667, 2015, 1012, 102],
```

Comment:
The ids changed here, is that an issue?

Reply:
before the mocked e5 model wasn't actually mocked but a small-ish e5 model. i have run our python tool to create a real mocked e5 model which i based on the `e5_small_v2` vocabulary. the old mocked e5 must have had a different base model/vocab, hence the changed values. in the end it doesn't matter for those tests as we mainly keep them for input/output size checks. but i checked that the new ids correspond to the correct words.